x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.tsx

```diff
@@ -75,7 +76,11 @@ const ConnectorsDropdownComponent: React.FC<Props> = ({
   selectedConnector,
   appendAddConnectorButton = false,
 }) => {
-  const { triggersActionsUi } = useKibana().services;
+  const {
+    triggersActionsUi,
+    application: { capabilities },
+  } = useKibana().services;
+  const canSave = hasSaveActionsCapability(capabilities);
```

Comment:
Do you mind if you use the Cases `useApplicationCapabilities` hook from `x-pack/plugins/cases/public/common/lib/kibana/hooks.ts`? You can use it like 

```
const { actions } = useApplicationCapabilities();
const canSave = actions.crud
```